01week/resume/Index resume.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>Ezequiel Gutierrez's Resume</title>
+ <link rel="stylesheet" href="./css/style.css">
+</head>
+  <img src="./desktop-image.jpg" alt= "Sun Set">
+<body>
+<header> <h1>Ezequiel Gutierrez.</h1> </header>
+
+<nav>
```

Comment:
Looks good! You definitely don't want to wrap each section in a <nav> tag. Remember that the tag should be reserved for the navigation links on your site https://www.w3schools.com/tags/tag_nav.asp